src/RFC-0190_Mempool.md

```diff
@@ -63,119 +80,118 @@ The Mempool is used for storing and managing unconfirmed [transaction]s.
-sharing the Mempool state with connected peers. An overview of the required functionality for the Mempool and each 
+The Mempool is responsible for managing, verifying and maintaining all unconfirmed transactions that have not yet
+been included in a [block] and added to the Tari [blockchain]. It is also responsible for propagating valid transactions and
+sharing the Mempool state with connected peers. An overview of the required functionality for the Mempool and each
 of its component pools will be provided.
 
 ### Overview
 
-Every base node maintains a Mempool that consists of two separate pools: the Unconfirmed Pool and Reorg Pool. 
-These two pools have different tasks and work together to form the Mempool used for maintaining unconfirmed transactions. 
+Every base node maintains a transaction pool called `Mempool` that consists of two separate pools: the Unconfirmed Pool and Reorg Pool.
+These two pools have different tasks and work together to form the Mempool used for maintaining unconfirmed transactions.
 
 This is the role descriptions for each component pool:
-- Unconfirmed Pool: contains all unconfirmed transactions that have been verified, have passed all checks, that 
-only spend valid [UTXO]s and don't have any time-lock restrictions.
-- Reorg Pool: stores a backup of all transactions that have recently been included into blocks, in case a blockchain 
-reorganization occurs and these transactions have to be restored to the Transaction Pool so that they can be included 
-in future blocks.
+- `Unconfirmed Pool`: contains all unconfirmed transactions that have been verified, have passed all checks, that
+  only spend valid [UTXO]s and don't have any time-lock restrictions.
+- `Reorg Pool`: stores a backup of all transactions that have recently been included into blocks, in case a blockchain
+  reorganization occurs and these transactions have to be restored back to the Unconfirmed Pool, so that they can be included
+  in future blocks.
 
 ### Prioritizing Unconfirmed Transactions
 
-The maximum storage capacity used for storing unconfirmed transactions by the Mempool and each of its component pools 
-can be configured. If a new transaction is received and the storage capacity limits have been reached, then 
-transactions are prioritized according to the transaction priority metric. The transaction priority metric consist 
-of the fee per gram of the UTXOs being spent by the transaction and should be applied to determine the priority 
-of each transaction in the Mempool. As the allocated storage space of the Mempool becomes limited, the transactions 
-with the highest priority are kept in the pool. The lowest priority transactions are discarded to make room for 
-higher priority incoming transactions.
+The maximum storage capacity used for storing unconfirmed transactions by the Mempool and each of its component pools
```

Comment:
I think we should first describe the tx priority algo in this section and then mention what happens if the storage capacity is exceeded.

https://github.com/tari-project/tari/blob/development/base_layer/core/src/mempool/priority/prioritized_transaction.rs#L41

Reply:
Actually, that might be overkill on the detail :smile: 

I think start with something like this
>transactions are prioritized by firstly the sum of fees of all UTXOs contained, followed by the length of time they have been in the mempool

and then mention what happens when the max storage cap is reached